FirebaseDynamicLinks/Sources/FIRDLJavaScriptExecutor.m

```diff
@@ -135,6 +150,21 @@ - (void)webView:(WKWebView *)webView
   [self handleExecutionError:error];
 }
 
+// Determine whether a process is running under Rosetta translation.
+// Returns 0 for a native process, 1 for a translated process,
+// and -1 when an error occurs.
+// From:
+// https://developer.apple.com/documentation/apple-silicon/about-the-rosetta-translation-environment
+int processIsTranslated() {
```

Comment:
Make this C function static so that it does not pollute the global symbol namespace.

Reply:
TIL... thanks!